tests/unit/src/kakarot/instructions/test_system_operations.py

```diff
@@ -71,8 +71,9 @@ async def test_call(
             contract_account_class.class_hash, account_registry.contract_address
         ).call()
 
+    @pytest.mark.parametrize("salt", [127, 256, 2**55 - 1])
```

Comment:
127 would fit in a single byte, meaning in rlp terms we wouldn't need a prefix.

256 and 2**55-1 both need prefixes